print-mptcp.c

```diff
@@ -182,6 +183,31 @@ struct mp_prio {
 
 #define MP_PRIO_B                       0x01
 
+static const struct tok mp_tcprst_flags[] = {
+        { 0x08, "U" },
+        { 0x04, "V" },
+        { 0x02, "W" },
+        { 0x01, "T" },
+        { 0, NULL }
+};
+
+static const struct tok mp_tcprst_reasons[] = {
+        { 0x06, "Middlebox interference" },
+        { 0x05, "Unacceptable performance" },
+        { 0x04, "Too much outstanding data" },
+        { 0x03, "Administratively prohibited" },
+        { 0x02, "Lack of resources" },
+        { 0x01, "MPTCP-specific error" },
+        { 0x00, "Unspecified error" }
```

Comment:
Thanks for the PR.
There is a missing '{ 0, NULL }' here.
And please rebase on top of the-tcpdump-group:master.

Reply:
> Thanks for the PR.
> There is a missing '{ 0, NULL }' here.

added, and ran again `make check` - to ensure it still passes when it parses MPTCP_TRST sub-option having `reason` equal to zero. 

> And please rebase on top of the-tcpdump-group:master.

done.
thanks for reviewing!
-- 
davide